{{cookiecutter.repostory_name}}/.envrc.template

```diff
@@ -15,4 +15,23 @@ export EMAIL_HOST=
 export EMAIL_PORT=
 export EMAIL_HOST_USER=
 export EMAIL_HOST_PASSWORD=
-export SENTRY_DSN={{cookiecutter.sentry_dsn}}
\ No newline at end of file
+export SENTRY_DSN={{cookiecutter.sentry_dsn}}
+export HTTPS_REDIRECT={{cookiecutter.https_redirect}}
+export HTTPS_PROXY_HEADER={{cookiecutter.https_proxy_header}}
+export CSP_ENABLED="{{cookiecutter.csp_enabled}}"
```

Comment:
It would be better to convert the 'y' and 'n' values to boolean before doing comparisions.

Reply:
What do you mean by boolean values? `0/1` or `true/false`?

Reply:
```
{% if cookiecutter.csp_enabled == 'y' %}
export CSP_ENABLED=True
{% else %}
export CSP_ENABLED=False
{% endif %}
```
and when using in settings.py
`CSP_ENABLED = env.bool('CSP_ENABLED')`

Reply:
Infact you can keep 
`export CSP_ENABLED={{ cookiecutter.csp_enabled }}`

because env.bool handles (y, n also).